pkgs/by-name/ph/phpdocumentor/package.nix

```diff
@@ -15,7 +16,7 @@ php.buildComposerProject (finalAttrs: {
     hash = "sha256-zpAKygpxyKYfefa5ag76saTSQNLon/v3rYkl0Nj2+FM=";
   };
 
-  vendorHash = "sha256-VNlAzWueF7ZXBpr9RrJghMPrAUof7f1DCh1osFIwFfs=";
+  vendorHash = "sha256-PJ5D48C1JThXblMlADm7YqI1iITsK/MYZW1fpkNr+54=";
```

Comment:
```suggestion
  vendorHash = "sha256-Kc88CPo+86EnnCyGW+u8hHx3qxvhGyASxwYiXlEneMc=";
```

Reply:
I run a build with empty hash and got different value `sha256-NfG69h0QIh/uOVi8Yi274ZK+sWvpEbiUrTd/ifStA7w=` in meantime. Maybe it's non-deterministic for some reason?

Reply:
Let me check again...

Reply:
Now I find `sha256-PJ5D48C1JThXblMlADm7YqI1iITsK/MYZW1fpkNr+54=`. Weird. There's something unstable somewhere...

Reply:
ofborg shows:
```
error: hash mismatch in fixed-output derivation '/nix/store/5qqkjhnkx2ninwpxfg20h0hy3a5nhb6z-phpdocumentor-3.5.3-composer-repository.drv':
         specified: sha256-PJ5D48C1JThXblMlADm7YqI1iITsK/MYZW1fpkNr+54=
            got:    sha256-L8QOA2O4lLf5y7oAkyB+ALaMRmwA9Cxsvw9x77OHtik=
```
and

```
error: hash mismatch in fixed-output derivation '/nix/store/flr9prqzn4lb52n31ps7zgbbq8yqjm3i-phpdocumentor-3.5.3-composer-repository.drv':
         specified: sha256-PJ5D48C1JThXblMlADm7YqI1iITsK/MYZW1fpkNr+54=
            got:    sha256-Kc88CPo+86EnnCyGW+u8hHx3qxvhGyASxwYiXlEneMc=
```

Reply:
There's something unstable, that's what I feared. We need to find the issue now...

Reply:
I quickly did this: 
![image](https://github.com/user-attachments/assets/28f6c2a9-c8e3-41f3-8f0f-2be2f1a31512)

But I will continue tomorrow... if you find anything in the meantime, open a PR to fix the issue in the builder.

Reply:
Maybe optimize autoload causes the issue